sdks/java/io/jdbc/build.gradle

```diff
@@ -33,6 +33,10 @@ dependencies {
   implementation library.java.joda_time
   implementation "org.apache.commons:commons-pool2:2.11.1"
   implementation library.java.slf4j_api
+  implementation 'org.postgresql:postgresql:42.6.0'
```

Comment:
I'm not sure we want to add all of these to the base jdbc package--I think the intent was for the user to provide their own dependencies. 

On the other hand, I can see it'd be really handy for cross-langauge users not to have to specify this. Perhaps these dependencies could be added to the expansion service instead?

Reply:
Yeah I wasn't sure the best spot to put them, but I agree putting them in the expansion service jar makes more sense.